styles.css

```
@@ -528,3 +528,28 @@ footer a:hover{text-decoration:underline}
   .org-logo{width:34px;height:34px}
   .org-name{font-size:12px}
 }
+
+/*style of score to top button*/
```

Stylelint (17.3.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)